Userland/test-web.cpp

```diff
@@ -360,7 +360,7 @@ JSFileResult TestRunner::run_file_test(const String& test_path)
             new_interpreter.run(new_interpreter.global_object(), *file_program.value());
 
             auto& before_initial_page_load = new_interpreter.get_variable("__BeforeInitialPageLoad__", new_interpreter.global_object()).as_function();
-            (void)new_interpreter.call(before_initial_page_load, JS::js_undefined());
```

Comment:
tsk tsk tsk, why can `undefined` be cast to an `Optional<MarkedValueList>` to begin with?

Reply:
nvm, am dumb.